dom.bs

```diff
@@ -6958,8 +6960,31 @@ a boolean <var>delegatesFocus</var>, and a boolean <var>slotAssignment</var>:
   <p>If <var>element</var> is a <a for=Element>shadow host</a>, then:
 
   <ol>
-   <li><p>If <var>element</var>'s <a for=Element>shadow root</a>'s <a for=ShadowRoot>declarative</a>
-   is false, then <a>throw</a> an "{{NotSupportedError!!exception}}" {{DOMException}}.
+   <li><p>Let <var>currentShadowRoot</var> be <var>element</var>'s
+   <a for=Element>shadow root</a>.
+
+   <li><p>If one of the following is true:
```

Comment:
This `<li>` contains multiple children so the `<p>` needs to be on a newline and indented. This also impacts all elements below which also do not look indented correctly.

Reply:
I *think* I addressed this comment, but let me know if I got something wrong.